### PR TITLE
feature/mx-1411 start configuring docker

### PR DESCRIPTION
**Pull request by cutoffthetop**
# PR Context
- still open
  - allow easy selectioni of which extractor to run with `make docker`
  - port over odbc config from singularity configuration

# Added
- add basic docker configuration with dockerfile, ignore and compose

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -33,6 +36,30 @@ wheel:
...
 
+image:
+	# build the docker image
+	@ echo building docker image mex-extractors:${LATEST}; \
+	export DOCKER_BUILDKIT=1; \
+	docker build \
+		--tag rki/mex-extractors:${LATEST} \
+		--tag rki/mex-extractors:latest .; \
+
+docker: image
```
Comment (mr-kamran-ali) [inline Makefile:47]: make docker and make image are big confusing I think. "make run" and "make image" would fit better.

File: compose.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+version: "3.8"
+services:
+  dagster-daemon:
+    build:
+      context: .
+    environment:
+      - DAGSTER_HOME=/app/work
+      - MEX_WORK_DIR=/app/work
```
Comment (mr-kamran-ali) [inline compose.yaml:8]: so we sticking with full path?

File: compose.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    entrypoint: ["dagster-daemon", "run", "--module-name", "mex"]
+    volumes:
+      - work:/app/work:rw
+  dagster-webserver:
+    build:
+      context: .
```
Comment (mr-kamran-ali) [inline compose.yaml:14]: and will it start same image twice with different entry points? is there a unified command from dagster to start both daemon and web server together?

**Review by mr-kamran-ali — COMMENTED** [review]
Ticket AC are not met, we can merge this but ticket will have to stay in TODO then.

File: Makefile (hunk 1 above)
Comment (cutoffthetop) [reply Makefile]: done 4c033d8

**Comment by cutoffthetop** [comment]
> Ticket AC are not met, we can merge this but ticket will have to stay in TODO then.

ticket updated ✔️

File: compose.yaml (hunk 2 above)
Comment (cutoffthetop) [reply compose.yaml:8]: yeah i think this works and is more expressive than just `./work`

File: compose.yaml (hunk 3 above)
Comment (cutoffthetop) [reply compose.yaml:14]: there is, that is what `dagster dev` does. but for deployment they recommend splitting that up; and the compose file is good for showing how someone could deploy our extractors

**Review by mr-kamran-ali — APPROVED** [review]
Awesome, need to merge main and then good to go.

**Pull request merged**